plugins/module_utils/network/sonic/utils/utils.py

```diff
@@ -379,6 +379,16 @@ def normalize_interface_name(configs, module, namekey=None):
                 conf[namekey] = get_normalize_interface_name(conf[namekey], module)
 
 
+def normalize_interface_name_list(configs, module):
```

Comment:
What is the reason for adding this function? (It apparently isn't being called by the new NTP code.)

Reply:
It is used to normalize source-interface list and called in config/ntp.py.

Reply:
Done

Reply:
I see it now. (I missed it earlier because of the elided diffs for that file.)